package.json

```diff
@@ -54,7 +54,6 @@
     "good-console": "^4.1.0",
     "graceful-fs": "^4.1.11",
     "hapi": "^8.4.0",
-    "hoek": "^2.12.0",
```

Comment:
No package lock changes after dropping this package ?

Reply:
@junedkazi yes, I double-checked.
Probably that’s because Hapi and its modules use "hoek" internally.